tests/compas/datastructures/test_halfedge.py

```diff
@@ -0,0 +1,95 @@
+import pytest
+
+from compas.datastructures.mesh.core import HalfEdge
+
+
+# ==============================================================================
+# Fixtures
+# ==============================================================================
+
+@pytest.fixture
+def halfedge():
```

Comment:
in my opinion, an instance of HalfEdge should be called mesh

Reply:
Don't know...but changed it to that 😄